examples/kubernetes/README.md

```diff
@@ -21,7 +21,7 @@ You'll need [Go 1.3+](http://golang.org/doc/install) in order to build the
 ### Build and install vtctlclient
 
 ```
-$ go get github.com/youtube/vitess/go/cmd/vtctlclient
+vitess/examples/kubernetes$ go build ../../go/cmd/vtctlclient
```

Comment:
Really?  I get the following:
# github.com/youtube/vitess/go/stats/influxdbbackend
../../gopath/src/github.com/youtube/vitess/go/stats/influxdbbackend/influxdb_backend.go:40: undefined: client.ClientConfig
../../gopath/src/github.com/youtube/vitess/go/stats/influxdbbackend/influxdb_backend.go:60: undefined: client.Series
../../gopath/src/github.com/youtube/vitess/go/stats/influxdbbackend/influxdb_backend.go:62: undefined: client.Series
../../gopath/src/github.com/youtube/vitess/go/stats/influxdbbackend/influxdb_backend.go:73: backend.client.WriteSeries undefined (type *client.Client has no field or method WriteSeries)